presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -231,7 +232,10 @@ protected void setup(Binder binder)
     {
         ServerConfig serverConfig = buildConfigObject(ServerConfig.class);
 
-        if (serverConfig.isCoordinator()) {
+        if (serverConfig.isResourceManager()) {
```

Comment:
This makes the resource manager and coordinator to not run on single server. Is that intentional? I think we should let it be configurable and not enforce it here.

Reply:
Can follow up on this in a later commit, for now given the experimental status of the component I don't think this is a must-need feature.